### PR TITLE
Pin rack to 1.6 so we can install passenger on ruby < 2.2

### PR DESCRIPTION
**Pull request by anfleene**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ end
 
 gem 'berkshelf', '~>3'
 gem 'test-kitchen'
+gem 'specinfra'
```
Comment (bbergstrom) [inline Gemfile]: Doesn't serverspec depend on specinfra?

File: test/integration/helpers/serverspec/Gemfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```
Comment (bbergstrom) [inline test/integration/helpers/serverspec/Gemfile]: test gemfile?

File: Gemfile (hunk 1 above)
Comment (anfleene) [inline Gemfile]: yeah I can delete that.

File: test/integration/helpers/serverspec/Gemfile (hunk 2 above)
Comment (anfleene) [inline test/integration/helpers/serverspec/Gemfile]: Yeah https://coderanger.net/serverspec-chef-11/

File: attributes/default.rb [hunk 3]
```diff
@@ -53,6 +53,7 @@
 
 # Passenger
 default[:passenger][:version] = "5.0.16"
+default[:passenger][:rack_version] = "1.6.4"
```
Comment (bbergstrom) [inline attributes/default.rb]: A comment about ruby 1.9 compatibility here would be good.  Someday we can remove it and update the gem path below for Ruby 2, the wave of the future.

File: test/integration/helpers/serverspec/Gemfile (hunk 2 above)
Comment (bbergstrom) [inline test/integration/helpers/serverspec/Gemfile]: :+1:

**Comment by bbergstrom** [comment]
:+1:

**Pull request merged**